src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -82,7 +82,10 @@ Status JobInfoAccessor::AsyncSubscribeAll(
         done(status);
       }
     };
-    RAY_CHECK_OK(AsyncGetAll(callback, /*timeout_ms=*/-1));
+    RAY_CHECK_OK(AsyncGetAll(callback,
+                             /*skip_job_info_field=*/false,
+                             /*skip_is_running_tasks_field=*/false,
```

Comment:
Can those be True as well If they are not used?

Reply:
Yes!